src/content/en/updates/2018/03/smooshgate.md

```diff
@@ -0,0 +1,217 @@
+
+- visitors of the affected websites suddenly get a broken user experience;
+- the website owners went from having a perfectly-working website to a
+  non-functional one without them changing anything;
+- browser vendors shipping the new feature lose market share, due to users
+  switching browsers after noticing “it works in browser X”;
+- once the compatibility issue is known, other browser vendors refuse to ship
+  it. The feature specification does not match reality ([“nothing but a work
+  of fiction”](https://www.webstandards.org/2009/05/13/interview-with-ian-hickson-editor-of-the-html-5-specification/#about-browsers)),
+  which is bad for the standardization process.
+
+Sure, in retrospect MooTools did the wrong thing — but breaking the web
+doesn’t punish them, it punishes users. These users do not know what a moo
+tool is. Alternatively, we can find another solution, and users can continue
+to use the web. The choice is easy to make.
+
+## Does that mean we can never remove bad APIs from the Web Platform? {: #removing-apis }
+
+It depends. In rare cases we can get away with [removing bad features from
+the Web]( https://whatwg.org/faq#removing-bad-ideas). Even just figuring
+out whether it’s _possible_ to remove a feature is a very tricky effort,
+requiring extensive telemetry to quantify how many web pages would have their
+behavior changed. But when the feature is sufficiently insecure, harmful to
+users, or is used very rarely, this can be done.
+
+`<applet>`, `<keygen>`, and
+[`showModalDialog()`](https://dev.opera.com/blog/showmodaldialog/) are all
+examples of bad APIs that were successfully removed from the Web Platform.
+
+## Why don’t we just fix MooTools? {: #fix-mootools }
+
+Patching MooTools so that it no longer extends built-in objects is a good
+idea. However, it doesn’t solve the problem at hand. Even if MooTools were
+to release a patched version, all existing websites using it would have to
+update for the compatibility problem to go away.
+
+Can’t people just update their copy of MooTools? {: #update-mootools }
+
+In a perfect world, MooTools would release a patch, and every single website
+using MooTools would magically be updated the next day. Problem solved,
+right?!
+
+Unfortunately, this is unrealistic. Even if someone were to somehow identify
+the full set of affected websites, manage to find contact information for
+each and every one of them, successfully reach out to all the website owners,
+and convince them all to perform the update (which might mean refactoring
+their entire code base), the entire process would take years, at best.
+
+Keep in mind that many of these websites are old and likely unmaintained.
+Even if the maintainer is still around, it’s possible they’re not a
+highly-skilled web developer like yourself. We can’t expect everyone to go
+and change their 8-year-old website because of a web compatibility issue.
+
+## How does the TC39 process work? {: #tc39 }
```

Comment:
The following paragraph just says it's a committee, not how it works.

Reply:
Good catch! The `#` at the start of the next paragraph turned it into a heading. Fixed.

Reply:
Ah. great. :)